src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.tsx

```diff
@@ -7,7 +7,7 @@ import { LEGACY_OPENCERTS_RENDERER } from "../../config";
 
 interface DecentralisedRendererProps {
   rawDocument: WrappedDocument;
-  updateTemplates: (templates: { id: string; label: string }[]) => void;
+  updateTemplates: (templates: { id: string; label: string; type: string }[]) => void;
```

Comment:
Maybe it's time to create a type... In the lib? Can start in the app first and move later (one day) in the lib